src/test/Components/CreateImageWizard/CreateImageWizard.compliance.test.js

```diff
@@ -124,8 +157,9 @@ describe('On Recreate', () => {
   test('with oscap profile', async () => {
     const user = userEvent.setup();
     await setup();
-
-    await screen.findByRole('heading', { name: /review/i });
+    screen.getByRole('button', {
+      name: /review/i,
+    });
```

Comment:
As most of the testing runs asynchronously the first option should be in my opinion be using the combination of `await` + `findBy` query.

The difference between `getBy` and `findBy` is that `getBy` tries to get an element instantaneously and fails if the element haven't been rendered yet. This could cause a certain level of flakiness in this case.

The combination of `await` and `findBy` is able to wait for the element to be rendered and therefore doesn't throw an error in case that didn't happen immediately.

Why did the two get switched here? Was the async `findBy` causing any problems in the tests?

Reply:
Oh yes! Good catch! This is super important to do.

Reply:
if I dont change it, the test fail    await screen.findByText('button', {
          |                  ^
      168 |       name: /review/i,
      169 |     });

Reply:
Hmm, it does seem to pass for me with this:

```suggestion
    await screen.findByRole('button', {
      name: /review/i,
    });
```

Wonder what the difference is. But also this shouldn't be a blocker I'd say. There should be some check and update of the tests anyway in the not so distant future as we are still using `getBy` in few other places.